aee.js

```diff
@@ -72,3 +72,13 @@ Array.prototype.count = function (spec) {
   }
   return result
 }
+
+Array.prototype.pluck = function (property) {
+  if (property === undefined) return null
```

Comment:
`property` is a mandatory arguments, we would probably better throw an exception.